readthedocs/storage/s3_storage.py

```diff
@@ -80,6 +69,28 @@ def __init__(self, *args, **kwargs):
         self.querystring_auth = False
 
 
+class S3StaticStorage(
+    S3StaticStorageMixin, OverrideHostnameMixin, S3ManifestStaticStorage, S3Boto3Storage
+):  # pylint: disable=too-many-ancestors
+
+    """
+    An AWS S3 Storage backend for static media.
+
+    * Uses Django's ManifestFilesMixin to have unique file paths (eg. core.a6f5e2c.css)
+    """
+
+
+class RTDS3StaticStorage(
```

Comment:
Don't know a better name for this

Reply:
```suggestion
class NoManifestS3StaticStorage(
```

Perhaps to be explicit?